packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -212,8 +213,9 @@ export class EpochTracker implements IPersistedFileCache {
         fetchOptions: {[index: string]: any},
         fetchType: FetchType,
         addInBody: boolean = false,
+        fetchReason?: string,
```

Comment:
For the future - if this signature keeps growing with more and more optional params, it might be nicer to use a params object instead.